graphics.p8

```diff
@@ -1,23 +1,13 @@
-pico-8 cartridge // http://www.pico-8.com
-version 18
-__lua__
--- pico8lib graphics library
--- by sparr
-
--- any function with a color parameter can add the following line
--- to make the color optional and use the pen color by default:
--- c = c or peek(24357) & 15
-
-
-------------------------------------------------------------------------
--- bresenham's line drawing algorithm
-
---- draw a line from point 1 to point 2
--- @tparam number x1 x coordinate of starting point
--- @tparam number y1 y coordinate of starting point
--- @tparam number x2 x coordinate of ending point
--- @tparam number y2 y coordinate of ending point
--- @tparam number c color
+--- @module graphics
+--- Drawing and sprite manipulation
+-- @author by sparr
+
+--- Bresenham's line drawing algorithm
+-- @param x1 Starting point of the line on the x axis
+-- @param y1 Starting point of the line on the y axis
+-- @param x2 Ending point of the line on the x axis
+-- @param y2 Ending point of the line on the x axis
+-- @param c The color of the line
```

Comment:
Why get rid of the tparam and types?

Reply:
ah, that was unintentional. I had documented this function locally and you had as well. When I merged from master I accidentally overwrote your changes. I will fix this.